components/src/utils/Cortex.js

```diff
@@ -62,9 +63,37 @@ export function cortexFetch(input, init) {
               type: '',
               id: '',
             };
+            const props = {};
+
             let debugMessages = '';
+            let debugMessageKey = '';
             for (let i = 0; i < data.messages.length; i++) {
-              debugMessages = debugMessages.concat(`${data.messages[i]['debug-message']} \n `);
+              const message = data.messages[i].data;
+              props.fieldName = message['field-name'];
+              props.maxSize = message.max;
+              props.minSize = message.min;
+              props.invalidValue = message['invalid-value'];
+              props.regExp = message['reg-exp'];
+              props.maximumNumberOfItems = message['maximum-number-of-items'];
+              props.GCCode = message['gift-certificate-code'];
+              props.couponCode = message['coupon-code'];
+              props.fieldValue = message['failed-value'];
+              props.userId = message['user-id'];
+              props.itemCode = message['item-code'];
+              props.minimumQuantity = message['minimum-quantity'];
+              props.shippingLevel = message['shipping-level'];
+              props.inventoryAvailable = message['inventory-available'];
+              props.quantityRequested = message['quantity-requested'];
+              if (data.messages[i].id === 'field.invalid.minimum.value') {
+                debugMessageKey = message['invalid-value'] ? data.messages[i].id : `${data.messages[i].id}-msg-2`;
+              } else if (data.messages[i].id === 'item.insufficient.inventory') {
+                debugMessageKey = message['quantity-requested'] ? `${data.messages[i].id}-msg-2` : data.messages[i].id;
+              } else if (data.messages[i].id === 'item.not.available') {
+                debugMessageKey = message['field-value'] ? `${data.messages[i].id}-msg-2` : data.messages[i].id;
+              } else if (data.messages[i].id === 'item.not.in.store.catalog') {
+                debugMessageKey = message['field-value'] ? `${data.messages[i].id}-msg-2` : data.messages[i].id;
+              } else debugMessageKey = data.messages[i].id;
+              debugMessages = debugMessages.concat(`${intl.get(debugMessageKey, props)} \n `);
```

Comment:
This is perfect - to use the message id as the key for our messages. If a message does not exist in our properties file, we should fallback to displaying the message as returned by Cortex.